ocaml/xenopsd/lib/xenops_task.ml

```diff
@@ -77,33 +77,13 @@ let parallel_id_with_tracing parallel_id t =
   | None ->
       parallel_id
 
-let traceparent_of_task t =
-  let ( let* ) = Option.bind in
-  let* tracing = Xenops_task.tracing t in
-  let* span = Tracing.Span.of_string tracing in
-  Tracing.Span.get_context span
-  |> Tracing.SpanContext.to_traceparent
-  |> Option.some
-
 let dbg_with_traceparent_of_task t =
   Option.fold ~none:(Xenops_task.get_dbg t)
     ~some:(fun traceparent -> Xenops_task.get_dbg t ^ "\x00" ^ traceparent)
-    (traceparent_of_task t)
+    (Xenops_task.tracing t)
```

Comment:
With this change it looks like we're missing a link from the parent, please confirm this is intended

Reply:
This is because xenopsd is now storing the traceparent header instead of the potentially bloated string representation of the Span.

Reply:
with this reduction the traceparent_of_task is redundent